Update Release List fails

### Steps to reproduce

This happened yesterday: 

https://github.com/vim/vim-win32-installer/runs/6870630744?check_suite_focus=true
```
The latest release in the list: v8.2.5076
Traceback (most recent call last):
  File "../main/scripts/update-release-list.py", line 124, in <module>
    main()
  File "../main/scripts/update-release-list.py", line 120, in main
    write_new_rels(new_rels, latest_rel)
  File "../main/scripts/update-release-list.py", line 83, in write_new_rels
    y = rel['published_at'][:4]
TypeError: 'NoneType' object is not subscriptable
New releases: v8.2.5085
Error: Process completed with exit code 1.
```

ping @k-takata 

### Expected behaviour

 no error

### Version of Vim

v8.2.5076

### Environment

Github Actions

--- a/scripts/update-release-list.py
+++ b/scripts/update-release-list.py
@@ -80,8 +80,11 @@ def write_new_rels(new_rels, latest_rel):
     print('New releases: ', end='')
     f = open(latest_file, 'w')
     for i, rel in enumerate(new_rels):
-        y = rel['published_at'][:4]
-        m = rel['published_at'][5:7]
+        pub_at = rel['published_at']
+        if not pub_at:
+            pub_at = rel['created_at']
+        y = pub_at[:4]
+        m = pub_at[5:7]
         if y != last_y:
             f.writelines(lines)
             f.close()

